reclaimer/sounds/sound_decompilation.py

```diff
@@ -20,14 +20,12 @@
 __all__ = ("extract_h1_sounds", "extract_h2_sounds", )
 
 
-def extract_h1_sounds(tagdata, tag_path, *,
-        out_dir="", write_wav=True, overwrite=True, decode_adpcm=True,
-        byteswap_pcm_samples=False, **kw):
-    do_write_wav = write_wav
-    overwrite = overwrite
-    decompress = decode_adpcm
-    pcm_is_big_endian = byteswap_pcm_samples
-    tagpath_base = os.path.join(out_dir, os.path.splitext(tag_path)[0])
+def extract_h1_sounds(tagdata, tag_path, **kw):
+    do_write_wav = kw.get('write_wav', True)
+    overwrite = kw.get('overwrite', True)
+    decompress = kw.get('decode_adpcm', True)
+    pcm_is_big_endian = kw.get('byteswap_pcm_samples', False)
+    tagpath_base = Path(kw['out_dir']).joinpath(Path(tag_path).with_suffix(""))
```

Comment:
No.

Reply:
Then we need to change all the other decompilation functions to have the used keyword args in the function args. I don't want some functions to do things one way and others doing it a different way. I've already said this, but consistency is important.

Reply:
I really do want to do that for almost every function

Reply:
We'll handle this later.

Reply:
Sounds good. As long as we handle them all at once, I have no objections.